process/views.py

```diff
@@ -232,6 +232,21 @@ def metadata(self, request, pk=None):
 
         return Response(metadata)
 
+    @action(detail=True)
+    def notes(self, request, pk=None):
+        """Return the notes for the compiled collection and its parent collection."""
+        compiled_collection = get_object_or_404(Collection, pk=pk)
```

Comment:
Why request the compiled collection? I think it'd be more generic to request the root collection, and then get notes from all child collections.

Reply:
I was following the same approach as the metadata endpoint, as I thought the registry would probably ask for the compiled collection. But you are right and I will change it to the root collection instead